src/components/widgets/gcode-preview/GcodePreview.vue

```diff
@@ -211,6 +228,14 @@ export default class GcodePreview extends Mixins(StateMixin) {
 
   panning = false
 
+  get isDelta (): boolean {
+    return this.$store.getters['printer/getPrinterSettings']('printer.kinematics') === 'delta'
```

Comment:
I believe your changes should work with both `delta` and `rotary_delta`, so can you make sure you check for both?